File: ts/iq/newsletter~get~status_updates.ts
/** <iq type="get" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError. */
export function buildNewslettersGetNewsletterStatusUpdates(to: string, to: string, count: number, since?: number): BinaryNode {
  return { tag: 'iq', attrs: { to: to, xmlns: 'newsletter', type: 'get' }, content: [
      { tag: 'status_updates', attrs: { count: count, since: since } },
    ] }
}

File: go/iq/newsletter~get~status_updates.go
// BuildNewslettersGetNewsletterStatusUpdates builds <iq type="get" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError.
func BuildNewslettersGetNewsletterStatusUpdates(to types.JID, to types.JID, count int, since int) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": to, "xmlns": "newsletter", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "status_updates", Attrs: waBinary.Attrs{"count": count, "since": since}},
		},
	}
}